snapshots/DeepSpeech/native_client/ctcdecode/third_party/openfst-1.6.7/src/include/fst/cache.h.snap

```diff
@@ -29101,14 +29103,14 @@ FuncSpace {
                             950,
                             115,
                             304,
-                            1187,
+                            1218,
                             682,
                             871,
                             1060,
-                            1218,
+                            1249,
                             414,
                             603,
-                            1249,
+                            1187,
```

Comment:
Why should a change of the line number position make `insta` fail here?

Reply:
@michaelvanstraten insta is comparing the whole output to the reference data, so if the output doesn't match it will fail